igor-web/src/test/groovy/com/netflix/spinnaker/igor/gitlabci/GitlabCiBuildMonitorSpec.groovy

```diff
@@ -105,21 +105,21 @@ class GitlabCiBuildMonitorSpec extends Specification {
         service.getProjects() >> [project]
         service.getPipelines(project, 5) >> [pipeline]
         buildCache.getJobNames(MASTER) >> jobsInCache
-        buildCache.getLastBuild(MASTER, "999", true) >> lastBuildNr
+        buildCache.getLastBuild(MASTER, "999", false) >> lastBuildNr
```

Comment:
Can you explain the change here?

Reply:
not in the sense that I could tell how it was working previously, but by setting `true` the code was avoiding the mock stubs set up for this test.

Reply:
Aaah, ok.  I see how that works when lastBuildNr is 0, but what about when it's 100?

Would we get verification of what's happening to move this to the `then:` section and make it
```suggestion
        1 * buildCache.getLastBuild(MASTER, "999", false) >> lastBuildNr
```
?